### PR TITLE
Fix sqlSrv odbc driver install issue

### PR DESCRIPTION
**Pull request by kpconnell**

### CODE REVIEW

**Comment by kpconnell** [comment]
Ya I understand..  but this extension needs that environment variable in
the container when it starts.  Using putenv during the test does not work.
If there is a generic way to inject environment variables into the tests
please let me know.

**Review by buddhaCode — APPROVED** [review]
I can approve these changes. Testing the layer locally with `layer=sqlsrv make test` succeeded. Also, I published the layer into my AWS account with `layer=sqlsrv php_versions=82 only_region=eu-central-1 make publish` and used it with my lambda functions. This also succeeded.

File: Makefile [hunk 1]
```diff
@@ -32,7 +32,7 @@ test: docker-images
 			echo "### Testing $${dir} PHP$${php_version}"; \
 			echo "###"; \
 			docker build --build-arg PHP_VERSION=$${php_version} --build-arg TARGET_IMAGE=$${dir}-php-$${php_version} -t bref/test-$${dir}-$${php_version} tests ; \
-			docker run --entrypoint= --rm -v $$(pwd)/$${dir}:/var/task bref/test-$${dir}-$${php_version} /opt/bin/php /var/task/test.php ; \
+			docker run --env ODBCSYSINI=/opt/microsoft/conf/ --entrypoint= --rm -v $$(pwd)/$${dir}:/var/task bref/test-$${dir}-$${php_version} /opt/bin/php /var/task/test.php ; \
```
[the remark this answers is not in the document]
Comment (buddhaCode) [reply Makefile:35]: I can confirm that removing this line would let the test fail.

**Comment by ddeboer** [comment]
Thanks for testing this @buddhaCode! Because this PR seems stalled, I created a cleaned-up and more minimal version at #479.

**Comment by buddhaCode** [comment]
Hahah :)
Testing it locally succeed.
But I couldn't publish the layer to my account:
```
php ./bref-extra publish
Found 1 new layers:
- sqlsrv-php-82

Publishing new layers:

Warning: file_get_contents(https://raw.githubusercontent.com/brefphp/bref/master/runtime/layers/regions.json): Failed to open stream: HTTP request failed! HTTP/1.1 404 Not Found
 in /Users/arne/Downloads/extra-php-extensions/src/Service/RegionProvider.php on line 12
make: *** [publish] Error 1
```

**Comment by Nyholm** [comment]
Closed by #479

**Pull request closed without merging**